data_model/src/account.rs

```diff
@@ -109,15 +101,17 @@ pub mod model {
 
     /// Builder which should be submitted in a transaction to create a new [`Account`]
     #[derive(
-        DebugCustom, Display, Clone, IdEqOrdHash, Decode, Encode, Serialize, Deserialize, IntoSchema,
+        DebugCustom, Display, Clone, IdEqOrdHash, Encode, Serialize, Deserialize, IntoSchema,
     )]
     #[display(fmt = "[{id}]")]
     #[debug(fmt = "[{id:?}] {{ signatories: {signatories:?}, metadata: {metadata} }}")]
     #[ffi_type]
+    #[serde(try_from = "candidate::NewAccount")]
     pub struct NewAccount {
         /// Identification
         pub id: AccountId,
         /// Signatories, i.e. signatures attached to this message.
+        /// Cannot be empty, guaranteed by constructors.
         pub signatories: Signatories,
```

Comment:
```suggestion
       signatories: Signatories,
```

can we do this. Also for `Account`? This is so that we maintain the invariant that account has to have at least one signatory